modules/_labs/slider/react/lib/Slider.tsx

```diff
@@ -0,0 +1,264 @@
+import * as React from 'react';
+import {
+  borderRadius,
+  colors,
+  spacingNumbers as spacing,
+  type,
+} from '@workday/canvas-kit-react-core';
+import {focusRing} from '@workday/canvas-kit-react-common';
+
+import styled from '@emotion/styled';
+
+export interface SliderProps {
+  max: number;
+  min: number;
+  startValue: number;
```

Comment:
I see this uses `startValue` instead of `value`. I think this should be a [controlled component](https://reactjs.org/docs/forms.html#controlled-components) like other form fields